### ISSUE TITLE
Slow application rendering (i.e. - slow resizing, slow preference dialog closing).

### ISSUE THREAD

**Issue by IndrekV** [toggl/toggldesktop#1471]
case 181332

**Comment by amulware** [comment]
For me the program is reasonably snappy (as much as I would expect from a winforms-wpf hybrid).

I will test this next week on a slower machine.

**Comment by amulware** [comment]
Tested on a 5 year old machine (Intel U3100 @ 1.3 ghz).
Performance is pretty bad. Most actions can take up to a second or two, especially the first time (opening auto complete, selection time entry, saving settings...)

**Comment by amulware** [comment]
Before we can do much about this, we need to:

- [ ] discuss what our performance goals are
- [ ] set up code to measure performance of a variety of systems
- [ ] collect data from a number of different platforms
- [ ] use that data to decide where optimizations are worthwhile.

**Comment by amulware** [comment]
This issue is on hold for now, until we are able to collect a lot more data.

Several improvements to the performance have already been done however, which might solve the problems for almost everybody.
